src/bin/generate_test_data.rs

```diff
@@ -7,25 +7,27 @@ use nmg_league_bot::models::season::{NewSeason, Season};
 
 extern crate dotenv;
 
+const PLAYERS_PER_BRACKET: i32 = 16;
+
 // Generates season, bracket, and 16 players, including 1 for me and 1 for my alt
 
 #[tokio::main]
-async fn main() {
```

Comment:
i ended up not needing to change anything in this file but i did some style cleanups while i was thinking about it anyway 🤷